### ISSUE TITLE
Subscribe only for specific events?

### ISSUE THREAD

**Issue by Ghostw4lk** [parse-community/ParseLiveQuery-Android#63]
Hey,

i have a question to subscribe only specific events [create, update...] with live query client.

For example, i only want get ParseObjects which are triggered by the CREATE-Event.
If i update an object on the server, the UPDATE-Event will be triggered.

My question is, is it possible to subscribe only for e.g. CREATE-Events?
Because all other events are not needed in my case and produce unnecessary traffic.

Thanks in adavance.

**Comment by flovilmart** [comment]
This is not supported at the moment and would likely require a refactor on parse-server. Do you want to open an issue on parse-server requesting that improvement please?

**Comment by Ghostw4lk** [comment]
Thanks for the quick response. 
I will open this issue in the parse-server repository.

**Comment by mmimeault** [comment]
On the client side, for now, you can only subscribe to a specific event and not get notified when it's other events. Events will still be received by the client (traffic) but not triggered.

**Comment by Ghostw4lk** [comment]
Yes, subscribe to specific events works well, but I think to avoid the traffic is necessary due to traffic costs.
